.env

```diff
@@ -0,0 +1,2 @@
+JWT_SECRET=adf.oiwrhaiuw98e6,8762371923uyiflkfbhw387;
```

Comment:
You forgot to add `.env` to your gitignore, your secret is out there 🔓 . You can add `*.env` to get rid of any w/ that file type.